README.md

```diff
@@ -476,7 +475,7 @@ Reference: https://github.com/facebook/jest/issues/708
 By default we use JSDOM v15, which requires Node v8+. If you want to use Node in a lower version to run your tests, make sure to configure jest to use an older JSDOM version, either in your `jest.config.js`, `jest.config.json` or `package.json`:
 
 ```json
-"testEnvironment": "jsdom"
+"testEnvironment": <jsdom_environment_package_<_15>
 ```
 
 If you use JSDOM v11 or lower, you might have to mock `localStorage` or `sessionStorage` on your own or using some third-party library by loading it in `setupFilesAfterEnv`.
```

Comment:
Can we remove paragraphs about JSDOM that are no longer necessary? Jest comes with JSDOM v15 now. If we want to guide users to use newer one, let's update the messaging.

Reply:
hmm I think indeed the paragraph is not necessary anymore. What I intended was to provide a fallback user guide but do we need a fallback anymore ? If we don't need, we can just remove the entire paragraph.

Reply:
On the other hand there's jsdom@16 that Jest 25 can't use, because it dropped Node 8. So the small reference can still be helpful to guide users too. But shouldn't be too lengthy

Reply:
I propose:

#### Configure other JSDOM versions

If you want to use another JSDOM version, either in your jest.config.js, jest.config.json or package.json:

```
"testEnvironment": <jsdom_environment_package>
```

Reply:
That's too vague, people will be confused. Let's use concrete example: https://www.npmjs.com/package/jest-environment-jsdom-sixteen

Reply:
Yeah, referencing the current version seems a good approach here.
Btw, if you could also remove the `testEnvironment` line from the exposed config in `README.md` would be awesome!

Reply:
I think it has been removed already :)

Reply:
Oh yeah, sorry about that!